arangodb-net-standard.Test/CollectionApi/CollectionApiClientTest.cs

```diff
@@ -103,5 +109,19 @@ public async Task PostCollectionAsync_ShouldThrow_WhenCollectionNameInvalid()
             });
             Assert.Equal(1208, ex.ApiError.ErrorNum);
         }
+
+        [Fact]
+        public async Task GetCollectionCountAsync_ShouldSucceed()
+        {
+            var newDoc = await _documentApi.PostDocumentAsync(_fixture.TestCollection, new PostDocumentsOptions());
+            var response = await _collectionApi.GetCollectionCountAsync(new GetCollectionCountOptions
+            {
+                CollectionName = _fixture.TestCollection
+            });
+
+            Assert.Equal(HttpStatusCode.OK, response.Code);
```

Comment:
What should happen if the collection name does not exist? Maybe add a test for that case? e.g. "GetCollectionCountAsync_ShouldThrow_WhenCollectionDoesNotExist"